roles/common/tasks/main.yml

```diff
@@ -1,4 +1,10 @@
 ---
+- name: common | Confirm that hostname matches inventory
+  ansible.builtin.hostname:
+    name: "{{ inventory_hostname }}"
```

Comment:
Is this going to add the `.princeton.edu` suffix to the hostname? If so, it would make for a long shell prompt.

Reply:
It add .princeton.edu to /etc/hostname, but not to the shell prompt thankfully!  I added the steps that @christinach and I used to exercise this branch to this PR's description, in case you'd like to try it out.